x-pack/docs/en/watcher/transform/script.asciidoc

```diff
@@ -39,25 +39,24 @@ The following table lists the possible settings that can be configured:
 
 [[transform-script-settings]]
 .Script Transform Settings
-[options="header,footer"]
+[options="header"]
 |======
 | Name      |Required | Default    | Description
 
-| `inline`  | yes*    | -          | When using an inline script, this field holds
+| `inline`  | yes    | -          | When using an inline script, this field holds
                                      the script itself.
 
-| `id`      | yes*    | -          | When referring to a stored script, this
+| `id`      | yes    | -          | When referring to a stored script, this
```

Comment:
I think these `|` were lining up before.

Reply:
I think you're right.  I've added a quick fix.